Makefile

```diff
@@ -16,57 +16,80 @@ TESTDATA := $(TOP_LEVEL)/test/data
 OS ?= linux
 ARCH ?= amd64
 BENCH_OUTPUT ?= stdout
+EXTENSIONS :=
```

Comment:
Can we not combine these two into just one variable?

EXTENSIONS :=
EXTENDED ?= sync,search,scrub,metrics,ui_base

like so:

EXTENSIONS ?= sync,search,scrub,metrics,ui_base

Reply:
Seems like a good idea. I will replace all occurrences of EXTENDED with EXTENSIONS,  which will require changes to created binary names.
Also binary-extended will disappear, replaced by binary ( which will be the equivalent of binary-extended, when variable EXTENSIONS isn't set )